src/ec/packages/ec-component-select/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@ecl-twig/ec-component-select",
+  "author": "European Commission",
+  "license": "EUPL-1.1",
+  "version": "0.0.1-alpha",
+  "description": "ECL Twig - EC Select",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@ecl-twig/ec-component-icon": "^0.0.1-alpha"
+  },
+  "devDependencies": {
+    "@ecl/ec-component-form-feedback-message": "^2.1.1",
+    "@ecl/ec-component-form-form-group": "^2.1.1",
+    "@ecl/ec-component-form-help-block": "^2.1.1",
+    "@ecl/ec-component-form-label": "^2.1.1",
+    "@ecl/ec-component-form-select": "^2.1.1",
+    "@ecl/ec-resources-icons": "~2.1.1",
```

Comment:
mixing `"~2.1.1"` and `"^2.1.1"` and (in other components) `"2.0.0"` or `"~2.1.0"`

we have to harmonize all that. let's discuss it on slack 😉